### PR TITLE
feat: update slo

### PR DESCRIPTION
**Pull request by KirillKurdyukov**

### CODE REVIEW

File: .github/workflows/slo.yml [hunk 1]
```diff
@@ -1,75 +1,80 @@
...
+
+      - name: Prepare SLO Database
+        run: |
+          cd slo/src/${{ matrix.sdk }}
+          dotnet run create grpc://localhost:2135 /Root/testdb
+
+      - name: Run SLO Tests
+        run: |
+          cd slo/src/${{ matrix.sdk }}
+          dotnet run run grpc://localhost:2135 /Root/testdb \
```
Comment (polRk) [inline .github/workflows/slo.yml:61]: run run?

Comment (KirillKurdyukov) [reply .github/workflows/slo.yml:61]: `dotnet run` run

File: .github/workflows/slo.yml [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
-    permissions:
-      checks: write
-      pull-requests: write
-      contents: read
-      issues: write
-
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sdk:
```
Comment (polRk) [inline .github/workflows/slo.yml:32]:
```suggestion
        workload:
```

File: .github/workflows/slo.yml [hunk 3]
```diff
@@ -1,75 +1,83 @@
...
-          timeBetweenPhases: 30
-          shutdownTime: 30
+          github_pull_request_number: ${{ github.event.inputs.github_pull_request_number }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          sdk_name: ${{ matrix.sdk }}
+          ydb_database_node_count: 5
+
+      - name: Prepare SLO Database
+        run: |
+          cd slo/src/${{ matrix.sdk }}
```
Comment (polRk) [inline .github/workflows/slo.yml:58]:
```suggestion
          cd slo/src/${{ matrix.workload }}
```

File: .github/workflows/slo.yml [hunk 4]
```diff
@@ -1,75 +1,83 @@
...
+          dotnet run create grpc://localhost:2135 /Root/testdb
+
+      - name: Run SLO Tests
+        run: |
+          cd slo/src/${{ matrix.sdk }}
```
Comment (polRk) [inline .github/workflows/slo.yml:63]:
```suggestion
          cd slo/src/${{ matrix.workload }}
```

File: slo/src/Internal/Cli.cs [hunk 5]
```diff
@@ -12,39 +12,19 @@ public static class Cli
         "db",
         "YDB database to connect to");
 
-    private static readonly Option<string> TableOption = new(
-        new[] { "-t", "--table-name" },
-        () => "testingTable",
-        "table name to create\n ");
+    private static readonly Option<string> PromPgwOption = new(
+        "--prom-pgw",
+        "minimum amount of partitions in table") { IsRequired = true };
```
Comment (polRk) [inline slo/src/Internal/Cli.cs:17]: its prometheus push gateway endpoint

File: slo/src/Internal/SloContext.cs [hunk 6]
```diff
@@ -91,12 +90,15 @@ PRIMARY KEY (hash, id)
 
     public async Task Run(RunConfig runConfig)
     {
+        // Trace.Listeners.Add(new ConsoleTraceListener()); debug meterPusher
+
         var promPgwEndpoint = $"{runConfig.PromPgw}/metrics";
         var client = await CreateClient(runConfig);
-        using var prometheus = new MetricPusher(promPgwEndpoint, Job, intervalMilliseconds: runConfig.ReportPeriod);
+        using var prometheus = new MetricPusher(promPgwEndpoint, "workload-" + Job,
```
Comment (polRk) [inline slo/src/Internal/SloContext.cs:97]: try just Job, without workload-prefix

**Review by polRk — APPROVED** [review]

**Pull request merged**